pyracing/response_objects/career_stats.py

```diff
@@ -0,0 +1,96 @@
+from ..constants import parse_iracing_string
+
+
+class CareerStats:
+    def __init__(self, dict):
+        self.wins = dict['wins']
+        self.winPerc = round(dict['winPerc'], 2)
+        self.poles = dict['poles']
+        self.totalclubpoints = dict['totalclubpoints']
+        self.avgStart = round(dict['avgStart'], 2)
+        self.avgFinish = round(dict['avgFinish'], 2)
+        self.top5Perc = round(dict['top5Perc'], 2)
+        self.totalLaps = dict['totalLaps']
+        self.avgIncPerRace = round(dict['avgIncPerRace'], 2)
+        self.avgPtsPerRace = round(dict['avgPtsPerRace'], 2)
+        self.lapsLed = dict['lapsLed']
+        self.top5 = dict['top5']
+        self.lapsLedPerc = round(dict['lapsLedPerc'], 2)
+        self.category = parse_iracing_string(dict['category'])
+        self.starts = dict['starts']
+
+
+class YearlyStats:
+    def __init__(self, dict):
+        self.wins = dict['wins']
+        self.winPerc = round(dict['winPerc'], 2)
+        self.year = dict['year']
+        self.poles = dict['poles']
+        self.clubpoints = dict['clubpoints']
+        self.avgStart = round(dict['avgStart'], 2)
+        self.avgFinish = round(dict['avgFinish'], 2)
+        self.top5Perc = round(dict['top5Perc'], 2)
+        self.totalLaps = dict['totalLaps']
+        self.avgIncPerRace = round(dict['avgIncPerRace'], 2)
+        self.avgPtsPerRace = round(dict['avgPtsPerRace'], 2)
+        self.lapsLed = dict['lapsLed']
+        self.top5 = dict['top5']
+        self.lapsLedPerc = round(dict['lapsLedPerc'], 2)
+        self.category = parse_iracing_string(dict['category'])
+        self.starts = dict['starts']
+
+
+class LastRacesStats:
+    def __init__(self, dict):
+        self.startPos = dict['startPos']
+        self.lapsLed = dict['lapsLed']
+        self.finishPos = dict['finishPos']
+        self.incidents = dict['incidents']
+        self.trackName = dict['trackName']\
```

Comment:
can we do `parse_iracing_string(dict['trackName'])` here?

Reply:
I've applied no string parsing yet. That can be in the another PR